include/mpm_base.h

```diff
@@ -195,6 +195,10 @@ class MPMBase : public MPM {
   unsigned quadrature_order_{2};
   // Level set methods
   bool ls_methods_{false};
+  //! Damping type
+  std::string damping_type_{""};
```

Comment:
Instead of a string use an Enum. In `MPMBase` define an enum

```
enum class Damping { None, Cundall };
```
Set the default to none